### ISSUE TITLE
K-12 users not receiving activation emails

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#454]
### Problem you are trying to address with this feature
In many school districts, children under the age of 16 are not allowed to send or receive email outside of the school or district’s domain. This presents a problem for K-12 students who sign up for our service, as our activation email will be blocked. Sometimes, we receive emails from teachers or school/district officials asking about this. Ideally, the school or district would allow Hypothesis (@hypothes.is) as a “trusted” or “allowed” domain before asking students to create their accounts. However, not all schools/districts allow for this, and even when they do, teachers don't always think to do this before having students create accounts. This means we need to manually activate accounts for students. 

### Your solution
Either:
- Provide an easy way to bulk-activate in the admin interface
- Provide an alternate means for self-activation

### User requests
- https://hypothesis.zendesk.com/agent/tickets/2141

**Comment by robertknight** [comment]
Hi @klemay - In addition to activation emails we also send emails for other purposes, such as notifying users about replies to annotations. Do these matter at all for these users or could we just live with these users not receiving these notifications?

If communication via email is not important then alternatively we could look at other ways to sign users up for accounts that don't rely on having an email address we can actually reach.

**Comment by klemay** [comment]
Good point @robertknight. From a support perspective, I don't personally have a need to communicate with users via email, but I don't know if I'm qualified to say whether it's important in general for the company.
